docs/object-detection.md

```diff
@@ -69,50 +69,67 @@ This convention is convenient and intuitive because it stores the config file fo
 <RVROOT>/processed-data
 ```
 
-## Setup
+## Running a test workflow locally
 
-Before running any commands, you will need to download some data.
-* Download the Mobilenet [pretrained model](http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_2017_11_17.tar.gz) and move the file to `<RVROOT>/pretrained-models/tf-object-detection-api/ssd_mobilenet_v1_coco_2017_11_17.tar.gz`.
-* Download the [ISPRS Potsdam](http://www2.isprs.org/commissions/comm3/wg4/2d-sem-label-potsdam.html) imagery using the [data request form](http://www2.isprs.org/commissions/comm3/wg4/data-request-form2.html) and place it in `<RVROOT>/raw-data/isprs-potsdam`.
-* Download the [cowc-potsdam labels](data/cowc-potsdam-labels.zip) and place them in `<RVROOT>/processed-data/cowc-potsdam/labels/`. These files were generated from the [COWC car detection dataset](https://gdo152.llnl.gov/cowc/) using scripts in [rv2.utils.cowc](../src/rv2/utils/cowc/).
+Before running the full workflow, you should check that the system is setup and running correctly by running a test workflow locally. This workflow uses a very tiny dataset so it will run quickly. First, you will need to copy files, edit configs, and run some commands.
 
-Next, you will need to copy and edit some config files.
-* Copy the Mobilenet [backend config files](../src/rv2/samples/backend-configs/tf-object-detection-api/) to `<RVROOT>/backend-configs/tf-object-detection-api/`.
-* Copy the [workflow config files](../src/rv2/samples/workflow-configs/object-detection/) to `<RVROOT>/workflow-configs/object-detection/`.
+* Copy the Mobilenet [model](http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_2017_11_17.tar.gz) that was pre-trained on COCO to `<RVROOT>/pretrained-models/tf-object-detection-api/ssd_mobilenet_v1_coco_2017_11_17.tar.gz`.
+* Copy the [test backend config file](../src/rv2/samples/backend-configs/tf-object-detection-api/mobilenet-test.config) to `<RVROOT>/backend-configs/tf-object-detection-api/mobilenet-test.config`.
+* Copy the [test data](https://github.com/azavea/raster-vision-data/releases/download/v0.0.1/cowc-potsdam-test.zip) and unzip it to `<RVROOT>/processed-data/cowc-potsdam-test`.
```

Comment:
Seeing `__MACOSX/` and `.DS_Store` files in the zip - probably should keep the the zips clean of these.

Reply:
Yea, I'll exclude them next time I make a release.